.github/workflows/macos-build.yml

```diff
@@ -0,0 +1,49 @@
+name: MacOS build
+on:
+  pull_request:
+
+jobs:
+  macos-build:
+    runs-on: macos-10.15
+    steps:
```

Comment:
need a check for `skip-ci`?

Reply:
I feel like this one should always run, but up to you.

Reply:
I also agree it's good to always run this.